### PR TITLE
Changes associated with renaming of 'solutionSample' to 'solution'

### PR DESCRIPTION
**Pull request by seenanair**
All 'solutionSample' fields are renamed to 'solution'

### CODE REVIEW

File: src/pages/registration/RegistrationForm.tsx [hunk 1]
```diff
@@ -278,15 +278,15 @@ const RegistrationForm = <T extends TissueValues<B>, B>({
                             )}
                           </FormikSelect>
                         )}
-                        {"solutionSample" in block && (
+                        {"solution" in block && (
                           <FormikSelect
                             emptyOption
                             label="Solution Sample"
```
Comment (khelwood) [inline src/pages/registration/RegistrationForm.tsx:284]: I think this should say "Solution"

File: src/lib/validation/registrationValidation.ts [hunk 2]
```diff
@@ -33,10 +33,10 @@ export default class RegistrationValidation {
     });
   }
 
-  get solutionSample() {
+  get solution() {
     return validation.requiredString({
       label: "Solution Sample",
```
Comment (khelwood) [inline src/lib/validation/registrationValidation.ts:38]: I think the label here should be Solution, not Solution Sample

File: src/pages/registration/RegistrationForm.tsx (hunk 1 above)
Comment (khelwood) [reply src/pages/registration/RegistrationForm.tsx:284]: Anywhere else too that says "solution sample"

Comment (seenanair) [reply src/pages/registration/RegistrationForm.tsx]: Sorry, I didn't get that.

Comment (seenanair) [reply src/pages/registration/RegistrationForm.tsx]: Changes done 👍

**Review by khelwood — APPROVED** [review]

**Pull request merged**